rest/admin_api.go

```diff
@@ -358,13 +358,13 @@ func (h *handler) handleActiveTasks() error {
 func (h *handler) handleGetRawDoc() error {
 	h.assertAdminOnly()
 	docid := h.PathVar("docid")
-	includeDoc := h.getBoolQuery("include_doc")
-	redact := h.getOptBoolQuery("redact", true)
+	includeDoc := h.getOptBoolQuery("include_doc", true)
+	redact := h.getBoolQuery("redact")
```

Comment:
Might as well switch to getOptBoolQuery("redact", false) to make the code quicker to parse.